Packs/SalesforceITAdmin/Integrations/Salesforce_IT_Admin/Salesforce_IT_Admin.py

```diff
@@ -0,0 +1,1164 @@
+import json
+import traceback
+
+import requests
+
+import demistomock as demisto
+from CommonServerPython import *
+
+''' IMPORTS '''
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' CONSTANTS '''
+URI_PREFIX = '/services/data/v44.0/'
+SCIM_EXTENSION_SCHEMA = "urn:scim:schemas:extension:custom:1.0:user"
+CUSTOM_MAPPING_CREATE = demisto.params().get('customMappingCreateUser')
+CUSTOM_MAPPING_UPDATE = demisto.params().get('customMappingUpdateUser')
+
+
+class Client(BaseClient):
+    """
+    Client will implement the service API, and should not contain any Demisto logic.
+    Should only do requests and return data.
+    """
+
+    def __init__(self, base_url, conn_client_id, conn_client_secret, conn_username,
+                 conn_password, headers, verify=True, proxy=False):
+        self.base_url = base_url
+        self.conn_client_id = conn_client_id
+        self.conn_client_secret = conn_client_secret
+        self.conn_username = conn_username
+        self.conn_password = conn_password
+        self.verify = verify
+        self.headers = headers
+        self.session = requests.Session()
+        if not proxy:
+            self.session.trust_env = False
+        res = self.create_login()
+        res_json = res.json()
+        self.headers['content-type'] = 'application/json'
+        if res_json.get('access_token') is not None:
+            self.headers['Authorization'] = "Bearer " + res_json.get('access_token')
+        else:
+            self.headers['Authorization'] = None
+
+    def http_request(self, method, url_suffix, params=None, data=None, headers=None):
+        if not headers:
+            headers = self.headers
+        full_url = self.base_url + url_suffix
+
+        res = requests.request(
+            method,
+            full_url,
+            verify=self.verify,
+            params=params,
+            json=data,
+            headers=headers
+        )
+
+        return res
+
+    def create_login(self):
```

Comment:
```suggestion
    def get_access_token(self):
```

Reply:
let's add here the logic to retrieve the access token instead of doing it after the function call in the c'tor

Reply:
Done the changes